yewtil/examples/function_component/Cargo.toml

```
@@ -8,7 +8,7 @@ edition = "2018"
 log = "0.4.8"
 wasm-logger = "0.2.0"
 yew = { path = "../../../yew" }
-yewtil = { path = "../.." }
+yewtil = { path = "../..", features = ["pure"]}
```

Shouldn't this be consistent with the other dependencies?
```suggestion
yewtil = { path = "../..", features = ["pure"] }
```